docs/data-structures/invert-binary-tree.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: Invert Binary Tree
+slug: /data-structures/invert-binary-tree
+---
+
+## Introduction
+
+An inversion, or mirror, of a Binary Tree ​ is just a Binary Tree  whose left and right children (of all non-leaf nodes) are swapped.
+
+## Explanation 
+
+The inverse of an empty tree is the empty tree. The inverse of a tree with root R, and subtrees T1 and T2, is a tree with root R, whose right subtree is the inverse of T1, and whose left subtree is the inverse of T2.
```

Comment:
The explanation isn't much explainable. Can you add a better explanation, with aid of text and pictures here?